topic:
  livox: "/livox/lidar"
  image: "/camera/image_color/compressed"
  img_compressed: true

common:
  visualize: true
  save: true
  save_img_path: "/home/psh/result.png"
  save_result_path: "/home/psh/result.txt"

calibration:
  camera_intrinsic: [861.96598, 0.0     , 622.33581,
                     0.0      , 865.0679, 532.89719,
                     0.0      , 0.0     , 1.0       ]
  dist_coeffs: [-0.236696, 0.082127, -0.001297, 0.000168, 0.000000] # k1, k2, p1, p2, k3

  # extrinsic parameter T_c_l
  extrinsic: [0.0062095,   -0.999969,  0.00476795, 0.008,
            -0.00404929, -0.00479315,    -0.99998, 0.065,
              0.999973 ,  0.00619007, -0.00407893, 0.05,
                    0.0,   0.0,    0.0,    1.0]
  
  # ???
  # camera_intrinsic: [852.304409, 0.0     , 628.399163,
  #                    0.0      , 852.304409, 497.286422,
  #                    0.0      , 0.0     , 1.0       ]
  # dist_coeffs: [-0.015718658, -0.000203527, 0.000170498, 8.90E-05, 1.17E-05] # k1, k2, p1, p2, k3

  # # extrinsic parameter T_c_l
  # extrinsic: [-0.040505, -0.998646, 0.032651, 0.016638,
  #             0.014399, -0.032091, -0.999381, 0.110230,
  #             0.999076, 0.040950, 0.013079, 0.010790,
  #             0.0, 0.0, 0.0, 1.0]
